packages/adaptive_scaffold/lib/src/adaptive_scaffold.dart

```diff
@@ -203,52 +237,182 @@ class AdaptiveScaffold extends StatefulWidget {
   /// list of [NavigationDestination]s. Takes in a [selectedIndex] property for
   /// the current selected item in the [NavigationRail] and [extended] for
   /// whether the [NavigationRail] is extended or not.
-  static Builder toNavigationRail({
+  static Builder toRailFromDestinations({
```

Comment:
Do we really need this method? The only thing it seems to save is mapping using `_toRailDestination`. If we really need it, shall we make a helper method that maps the destinations and converts `_toRailDestination`, so that developers can use it _on top of_ `toNavigationRail`?

Reply:
I also agree! I ended up making a public helper function in the adaptive scaffold file. Let me know if I need to change that!